src/Files.App.BackgroundTasks/Files.App.BackgroundTasks.csproj

```diff
@@ -1,20 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
     <PropertyGroup>
         <TargetFramework>net7.0-windows10.0.22621.0</TargetFramework>
-        <AssemblyName>BackgroundTasks</AssemblyName>
```

Comment:
I remember you concerned if changing this will cause some problems or not.

I already renamed it in Package.appmanifest as well and there's no problem.